core/templates/pages/exploration-player-page/new-lesson-player/new-lesson-player-components/player-sidebar.component.html

```diff
@@ -1,4 +1,111 @@
-<div class="oppia-lesson-player-sidebar"
-     [class.expanded]="isExpanded"
-     (click)="toggleSidebar()">
+<div class="oppia-lesson-player-sidebar">
+  <div class="player-desktop-sidebar-menu"
+       [class.expanded]="isExpanded">
+    <div class="lesson-player-sidebar-top">
+      <button class="player-sidebar-toggle"
+              (click)="toggleSidebar()"
+              [class.expanded]="isExpanded">
+        <div *ngIf="!isExpanded">
+          Open options
+          <img src="/assets/images/lesson_player/sidebar-chevron-right.svg" alt="Open options">
+        </div>
+        <div *ngIf="isExpanded">
+          Close options
+          <img src="/assets/images/lesson_player/sidebar-chevron-left.svg" alt="Close options">
+        </div>
+      </button>
+      <div *ngIf="isExpanded">
+        <div class="oppia-exploration-description"
+            *ngIf="!isHackyExpDescTranslationDisplayed()">
+          {{ expDesc | truncateAndCapitalize: 90 }}
+        </div>
+        <div class="oppia-exploration-description"
+             tabindex="0"
+             *ngIf="isHackyExpDescTranslationDisplayed()">
+          {{ expDescTranslationKey | translate | truncateAndCapitalize: 90 }}
+        </div>
+
+        <span class="player-sidebar-rating" [ngClass]="{'rating-disabled': !avgRating}">
+          <span *ngIf="avgRating" class="player-rating-text">
+            {{ avgRating | number:'1.1-1' }}
+          </span>
+          <span *ngIf="!avgRating"
+                [innerHTML]="'I18N_LIBRARY_N/A' | translate"
```

Comment:
Why is there an N/A here? I didn't really understand that bit when watching the videos.

Also is this the average rating of the lesson or the rating given by the learner?

A more general thought: if we're unsure about this bit, I wonder if we could drop the rating display for now, and leave it for later. I feel a bit uncertain about why it needs to be shown, and left a question for the designer in the mocks (though if you have any thoughts, feel free to respond to it! -- e.g. there might be some clarification on this in a place I haven't looked at yet!).

Reply:
Well I just mimicked what the current exploration rating display already has - displaying N/A when something has been unrated.

Also it is the rating of the lesson, calculated by reusing current services. Let me know if we choose to drop the rating display completely, although the sidebar might look a bit empty then.

Reply:
Ah ok, thanks. Yes, I think that works in the community library, but we don't do that in the exploration player currently, right?

And yeah, let's drop the rating display completely for now and get this PR merged. I'll let you know if we need to put it (or a version of it) back in again, but I think the discussions with the designer might take a little while, so best not to block on it.

Thanks!

Reply:
We do that in both, the community library and the exploration player for every single lesson currently. In the lesson info card for the current exploration player. I'll drop the display for now, we can just add it back if needed.